index.html

```diff
@@ -14,9 +15,22 @@
                             <grow>
                                 <gcol>
                                     <input id="cardName" placeholder="card"></input>
+                                    <section>
+                                        <article class="title">
+                                        </article>
+                                    </section>
+                                    <section>
+                                        <article class="descr">
+                                        </article>
+                                    </section>
                                 </gcol>
                                 <gcol id="cardButnCell">
-                                    <button id="cardButn"></button>
+                                    <button id="cardButn">
+                                        <i class="icon-plus"></i>
+                                    </button>
+                                    <button id="cardToss">    
+                                        <i class="icon-delete"></i>
+                                    </button>
```

Comment:
The delete button should be in the card (in the javascript when each card is created/refreshed from memory) .. not here.